src/pages/{mdx.frontmatter__permalink}.js

```diff
@@ -48,7 +75,7 @@ const BlogPost = ({ data }) => {
   
   return (
     <div onScroll={handleScroll}>
-      <SocialBlog className={disappearSocial ? styles.socialDisappear : styles.socialAppear} />
+      <SocialElement className={classnames(disappearSocial ? styles.socialDisappear : styles.socialAppear, styles.blogIcons)} links={shareBlogPostLinks} />
```

Comment:
What does styles.socialDisappear do? If it only hide the SocialElement you can use `disappearSocial && <SocialElement ...`

Reply:
Can you explain it in another way? Please.